src/structures/guild.ts

```diff
@@ -133,16 +133,11 @@ export class GuildBans extends Base {
    * @param user User to unban, ID or User object.
    */
   async remove(user: string | User): Promise<boolean> {
```

Comment:
Supports audit reason btw https://discord.com/developers/docs/resources/guild#remove-guild-ban

Reply:
kay imma add it too